plugins/inventory/gql_inventory.py

```diff
@@ -67,7 +67,8 @@
             - List of dot-sparated paths to index graphql query results (e.g. `platform.slug`)
             - The final value returned by each path is used to derive group names and then group the devices into these groups.
             - Valid group names must be string, so indexing the dotted path should return a string (i.e. `platform.slug` instead of `platform`)
-            - If value returned by the defined path is a dictionary, an attempt will first be made to access the `name` field, and then the `slug` field. (i.e. `platform` would attempt to lookup `platform.name`, and if that data was not returned, it would then try `platform.slug`)
+            - If value returned by the defined path is a dictionary, an attempt will first be made to access the `name` field, and then the `slug` field.
+              (i.e. `platform` would attempt to lookup `platform.name`, and if that data was not returned, it would then try `platform.slug`)
```

Comment:
Does this actually work? Not sure if it would throw some kind of syntax error either running the inventory or attempting to build docs.

Reply:
Looks like it did. I made an update regardless.

```
❯ hacking/make-docs.sh
rm: cannot remove 'ansible_collections': No such file or directory
rm: cannot remove 'tests/output': No such file or directory
rm: cannot remove '.pytest_cache': No such file or directory
Using /home/joshv/projects/nautobot-ansible/ansible.cfg as config file
Created collection for networktocode.nautobot at /home/joshv/projects/nautobot-ansible/networktocode-nautobot-3.4.0.tar.gz
Starting galaxy collection install process
Process install dependency map
Starting collection install process
Installing 'networktocode.nautobot:3.4.0' to '/home/joshv/.ansible/collections/ansible_collections/networktocode/nautobot'
networktocode.nautobot (3.4.0) was installed successfully
```